### PR TITLE
Improve design using css

### PR DESCRIPTION
**Pull request by cristyjoy**

### CODE REVIEW

File: joy/myblog/myblogspot/models.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    sub_Heading = models.CharField(max_length=150)
+
+    def __str__(self):
+        return '{}'.format(self.heading)
+
+class Post(models.Model):
+    STATUS_CHOICES = (('publishsed', 'Published'), ('draft', 'Draft'), ('hidden', 'Hidden'),)
+    user = models.ForeignKey(User,on_delete=models.CASCADE)
+    title = models.CharField(max_length=150)
+    sub_Title = models.CharField(max_length=150)
```
Comment (rocity) [inline joy/myblog/myblogspot/models.py]: @cristyjoy field names should be in snake case

File: joy/myblog/myblogspot/models.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    banner_Photo = models.ImageField(upload_to = 'static/media')
```
Comment (rocity) [inline joy/myblog/myblogspot/models.py]: @cristyjoy field names should be in snake case

File: joy/myblog/myblogspot/models.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    category = models.ForeignKey('Category', on_delete=models.CASCADE)
+    tags = models.ManyToManyField('Tag',related_name="Post")
+    status = models.CharField(max_length=10, choices=POST_STATUS, default='draft')
+
+
+    def __str__(self):
+        return '{}'.format(self.title)
+
+class Category(models.Model):
+    category = models.CharField(max_length=150)
```
Comment (rocity) [inline joy/myblog/myblogspot/models.py]: @cristyjoy what's this field for?

File: joy/myblog/myblogspot/models.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+    def __str__(self):
+        return '{}'.format(self.category)
+
+class Tag(models.Model):
+    tags = models.CharField(max_length=150)
```
Comment (rocity) [inline joy/myblog/myblogspot/models.py]: @cristyjoy what's this field for?

File: joy/myblog/templates/registration/home.html [hunk 5]
```diff
@@ -0,0 +1,5 @@
+ <!-- begin wwww.htmlcommentbox.com -->
```
Comment (rocity) [inline joy/myblog/templates/registration/home.html]: @cristyjoy what's this?

File: joy/myblog/templates/registration/password_reset_confirm.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
+{% extends "base_generic.html" %}
```
Comment (rocity) [inline joy/myblog/templates/registration/password_reset_confirm.html]: @cristyjoy make this file's indents uniform

Comment (cristyjoy) [reply joy/myblog/templates/registration/password_reset_confirm.html]: mag ulit sa ko sir kay dli nko ka migrate

**Pull request merged**